### ISSUE TITLE
[Request] In combat no fade, out of combat fade

### ISSUE THREAD

**Issue by benoitheylens** [kesava-wow/kuinameplates2#89]
Hi there,

Great addon ! I really love what you did with names-only mode.

Here is a suggestion, unfortunately I did not achieve to configure it on my own, maybe there is a way to do so, but I didn't find it.

I just want to display **at all time**, combat and out of combat, enemy plates.
When I enter combat, there is only the target that is displayed with no fading. It's very annoying and confusing, we can't follow what happen to other target (stuns, cc, dots) ...
I did configure my KUi AND my wow UI to always display names but still not the results I wanted.

To make things clear, here is two screenshot of my current situation :
![wowscrnshot_090616_202735](https://cloud.githubusercontent.com/assets/21984111/18290916/b24a4d10-7485-11e6-9e6f-2994b4722ca6.jpg)

![wowscrnshot_090616_202730](https://cloud.githubusercontent.com/assets/21984111/18290917/b24b22ee-7485-11e6-81e7-56f84c7be6bf.jpg)

Wow config : display all nameplates, display ally names and enemy names, CTRL+V : all plates activated.

KUI config : 
```
KuiNameplatesCoreSaved = {
	["profiles"] = {
		["default"] = {
			["fade_alpha"] = 0,
			["font_size_small"] = 12,
			["nameonly_target"] = false,
			["execute_colour"] = {
				1, -- [1]
				1, -- [2]
				1, -- [3]
			},
			["font_size_normal"] = 14,
			["target_glow_colour"] = {
				1, -- [1]
				0.501960784313726, -- [2]
				0, -- [3]
				0.6, -- [4]
			},
			["castbar_height"] = 6,
			["colour_player"] = {
				0.23921568627451, -- [1]
				0.4, -- [2]
				0.705882352941177, -- [3]
			},
			["fade_avoid_execute_hostile"] = true,
			["fade_avoid_execute_friend"] = true,
			["execute_auto"] = false,
			["classpowers_size"] = 12,
			["fade_untracked"] = true,
			["colour_neutral"] = {
				0.792156862745098, -- [1]
				0.6, -- [2]
				0, -- [3]
			},
			["bar_texture"] = "Minimalist",
		},
	},
}

```

```
KuiNameplatesCoreSaved = {
	["profiles"] = {
		["default"] = {
			["fade_alpha"] = 0,
			["font_size_small"] = 12,
			["nameonly_target"] = false,
			["execute_colour"] = {
				1, -- [1]
				1, -- [2]
				1, -- [3]
			},
			["font_size_normal"] = 14,
			["target_glow_colour"] = {
				1, -- [1]
				0.501960784313726, -- [2]
				0, -- [3]
				0.6, -- [4]
			},
			["castbar_height"] = 6,
			["colour_player"] = {
				0.23921568627451, -- [1]
				0.4, -- [2]
				0.705882352941177, -- [3]
			},
			["fade_avoid_execute_hostile"] = true,
			["fade_avoid_execute_friend"] = true,
			["execute_auto"] = false,
			["classpowers_size"] = 12,
			["fade_untracked"] = true,
			["colour_neutral"] = {
				0.792156862745098, -- [1]
				0.6, -- [2]
				0, -- [3]
			},
			["bar_texture"] = "Minimalist",
		},
	},
}
```
tldr; I want that the configuration I get out of combat stays the same in combat.

**Comment by kesava-wow** [comment]
Thanks for the detail. It's a breath of fresh air, honestly. 

Anyway, you have /knp > general > faded alpha set to 0. This means when you have a target, everything else gets faded out. You want to set to... Well, anything but 0.

**Comment by benoitheylens** [comment]
Well ... I think you missed my point.

I configured the addon so I am very pleased how it works out of combat.
WHY does it only shows the target enemy nameplate healthbar in combat mode ? As it's configured, it's perfect out of combat. I just want consistency when I enter combat and show every enemy healthbars.

I want to see friendly names like I did configure them in WoW (only quest NPC and enemies), even in combat (and especially in combat). It was perfect back when "names-only" did not exist.

**Comment by kesava-wow** [comment]
> WHY does it only shows the target enemy nameplate healthbar in combat mode

I answered this. Because you have "faded alpha" set to 0. But I understand you have it set to 0 in order to hide friendly frames. Like I said, that is a distinct issue: #75.

**Comment by benoitheylens** [comment]
Ok, thanks for the explanation. I just don't understand why it changes when I enter combat. Is this something blizzard-related ? 
Anyway, once your secondary value will be up, I won't never, ever go back to anything else but your addon. I convinced most of my guild to use it ;).
